Homework3

FrankieChik opened the pull request:
I try to finish the assignment but still get so many things unsolved. I may need to ask you in person the parts I am troubled about.

Commit:
Homework2.ipynb

Commit:
Homework2(This_is_my_answer).ipynb

Commit:
Homework2Finalanswers.ipynb

jdamerow committed:
homework assignment 6

Commit:
03Homeworkanswers.ipynb

jdamerow commented on 03Homeworkanswers.ipynb:
```diff
@@ -0,0 +1,117 @@
+    "    if not item in book:\n",
+    "        print(\"You can request our library to buy \"+ item)\n",
+    "    elif item in otherbook:\n",
+    "        print(\"We have his otherbooks.\")"
```
This doesn't implement the things asked for in the assignment:
- if a book is not in the library, users should be able to request them to be bought
- users can also donate books to the library
- the library also stores the authors of books and users can query for those (e.g. enter a title and the program returns the authors)

FrankieChik committed:
Merge pull request #11 from diging-training/master

homework assignment 6

jdamerow committed:
Added homework for week 7

Commit:
Homeowrk4.ipynb

FrankieChik committed:
Merge pull request #14 from diging-training/master

Added homework for week 7

jdamerow committed:
added homework for week 8

FrankieChik committed:
Create README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

FrankieChik committed:
Update README.md

Commit:
Homework-05.ipynb

FrankieChik committed:
Merge pull request #16 from diging-training/master

added homework for week 8

FrankieChik committed:
Create index.md

FrankieChik committed:
Create secondfile

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update index.md

FrankieChik committed:
Update secondfile

Commit:
03Homework-answers.ipynb

Commit:
Homework2(This_is_my_answers).ipynb

Commit:
Homework-06-documentation-testsanswers.ipynb

FrankieChik committed:
Create project.py

FrankieChik committed:
Update project.py

FrankieChik committed:
Create project_code_04082020.py

Hi Julia and Ken,
I went over the program historians and other online topic modeling resources these two days. I am finally able to tokenize and clean the texts I am now working on. I ran the code once on Pycharm but only it came out with some numbers like [(0, 24), (1, 21), (2, 6), (3, 3), (4, 3), (5, 3), (6, 3), (7, 3), (8, 3), (9, 3), (10, 3), (11, 3), (12, 3).......
I think I may need to process topic modeling this week but may I know if I am on the right track? Thanks.
